src/components/bs5/footer/footer.hbs

```diff
@@ -7,9 +7,8 @@
                 <h2 class="footer-site-name">{{ sitename }}</h2>
             </div>
         </div>
-
-        <div class="row">
-            
+        
+        <div class="row {{#unless followlinksDisabled}}{{#if feedbackFormDisabled}}r25{{else}}r20{{/if}}{{else}}{{#if feedbackFormDisabled}}r33{{else}}r25{{/if}}{{/unless}}">
```

Comment:
could we look at changing col-xs-12 col-md-12 col-lg-3 these line instead of doing a custom css for it.